server/api/donations.js

```diff
@@ -0,0 +1,47 @@
+const router = require('express').Router()
+const { DonationProduct } = require('../db/models')
+module.exports = router;
+
+// /donations - shows all donation products
+// /donations/:id - shows a particular donation product
+// /donations/new - creates a donation product
+// /donations/:id/update - updates a donation product
+// /donations/:id/delete - deletes a donation product
+// /donations/:id/addReview for a donation product
+
+// GET /api/donations
+router.get('/', (req, res, next) => {
+  Donation.findAll()
+    .then(donations => res.json(donations))
+    .catch(next);
+})
+// GET /api/donations/:donationId
+router.get('/:donationId', (req, res, next) => {
+  const id = req.params.donationId;
+  Donation.findById(id)
+    .then(donation => res.json(donation))
+    .catch(next);
+})
+
+// POST /api/donations/newDonation
+router.post('/newDonation', (req, res, next) => {
```

Comment:
If you want this to be a RESTful app, I would just POST to '/' (aka. '/api/donations'). But this is preference, just something I thought I would point out